### PR TITLE
Update UML Diagrams in DG

### PR DESCRIPTION
**Pull request by bchenghi**
Update UML Diagrams

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -107,10 +107,12 @@ The `Model`,
 * does not depend on any of the other three components.
 
 
-<div markdown="span" class="alert alert-info">:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Person` references. This allows `AddressBook` to only require one `Tag` object per unique `Tag`, instead of each `Person` needing their own `Tag` object.<br>
+<div markdown="span" class="alert alert-info">
+:information_source: **Note:** An alternative (arguably, a more OOP) model is given below. It has a `Tag` list in the `AddressBook`, which `Person` references. This allows `AddressBook` to only require one `Tag` object per unique `Tag`, instead of each `Person` needing their own `Tag` object.<br>
```
Comment (songyiang) [inline docs/DeveloperGuide.md]: AddressBook references have not been removed.

**Review by songyiang — COMMENTED** [review]
Just one minor comment.

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (bchenghi) [reply docs/DeveloperGuide.md]: Okay changed. Thanks.

**Review by songyiang — APPROVED** [review]
LGTM

**Pull request merged**